gui/default/syncthing/folder/editFolderModalView.html

```diff
@@ -11,11 +11,22 @@ <h4 class="modal-title">
         <form role="form" name="folderEditor">
           <div class="row">
             <div class="col-md-12">
+              <div class="form-group" ng-class="{'has-error': folderEditor.folderLabel.$invalid && folderEditor.folderLabel.$dirty}">
+                <label for="folderLabel"><span translate>Folder Label</span></label>
+                <input name="folderLabel" id="folderLabel" class="form-control" type="text" ng-model="currentFolder.label" ng-if="editingExisting"/>
+                <input name="folderLabel" id="folderLabel" class="form-control" type="text" ng-model="currentFolder.label" ng-if="!editingExisting &&  currentFolder.viewFlags.importFromOtherDevice" value="{{currentFolder.label}}"/>
```

Comment:
I can not know, if I'm *not* editing an existing entry, if I import an share-request from another device or if I'm adding a new folder from the local machine.

This is a direct consequence of 
> When adding a new folder from a share request, prefill both folder ID and label, allowing label to be edited at will (but not ID as it would not make sense to do so at that point)